spec/infinispan_ssl_spec.js

```diff
@@ -31,6 +35,14 @@ describe('Infinispan TLS/SSL client', function() {
      testError('CERT_SIGNATURE_FAILURE', sslSniUntrusted())
   );
 
+  it('fails to operate if server name (SNI) is not provided, but certificate is trusted',
+      testError('CERT_SIGNATURE_FAILURE', sslSniDefaultWithTrustedCertificate())
+  );
+
+  it('fails to operate if server name (SNI) and trusted certificate are incorrect',
```

Comment:
Is this description representative of what the test does? The client does not seem to provide any trusted certificates here.